### PR TITLE
Camera name, camera name in admin list, translations.

### LINKED ISSUE #81: Add camera device name
This device name will be used in authentication. It should follow some regular expression. This name has to be unique among all device names.

### LINKED ISSUE #178: Admin: Improve camera list
* Show camera name in the list (blocked by #81)
  * Add translations for column names
* Update `api.models.Camera.__str__`

### PR DESCRIPTION
**Pull request by Rhantolq**
Added camera name with regex check.
Camera name is displayed in list admin view.

Closes #81 and closes #178

### CODE REVIEW

File: django-server/server/api/serializers.py [hunk 1]
```diff
@@ -26,9 +26,11 @@ class Meta:
 
 
 class CameraSerializer(serializers.ModelSerializer):
+    name = serializers.RegexField('^[a-zA-Z0-9-_]+$')
+
     class Meta:
         model = Camera
-        fields = ['id', 'state', 'cafeteria']
+        fields = ['id', 'state', 'name', 'cafeteria']
```
Comment (kantoniak) [inline django-server/server/api/serializers.py:33]: Could you change order to put name second?

**Comment by Rhantolq** [comment]
@avan1235  it is as Krzysiu says: the numbers are for programmers convenience. They are generated by django-admin makemessages

File: django-server/server/api/serializers.py (hunk 1 above)
Comment (avan1235) [reply django-server/server/api/serializers.py:33]: > It's okay, Django generates this file and changes lines numbers to match existing code.

Thank's for explanation

**Comment by avan1235** [comment]
> @avan1235 it is as Krzysiu says: the numbers are for programmers convenience. They are generated by django-admin makemessages

Thank's for explanation

**Pull request merged**